torch/testing/_internal/common_methods_invocations.py

```diff
@@ -20322,6 +20326,16 @@ def __init__(
             DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_python_ref_executor'),
         ),
     ),
+    ElementwiseUnaryPythonRefInfo(
+        "_refs.nn.functional.threshold",
+        torch_opinfo_name="nn.functional.threshold",
+        skips=(
+            # NotImplementedError: Could not run 'aten::where.ScalarOther' with arguments from the 'AutogradMeta'...
+            DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_python_ref_meta',),
+            # RuntimeError: Tracing expected 3 arguments but got 1 concrete arguments
+            DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_python_ref_executor'),
```

Comment:
@mruberry is it expected that I need to xfail this test? 

For the ref_meta one i think there is a missing meta kernel, but for this one i'm not sure what the reason is.

Reply:
This should be fixed after https://github.com/pytorch/pytorch/pull/79858 from @ngimel, but yes its' fine to xfail this for now